### PR TITLE
cgclassify: add page

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/cgclassify.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# cgclassify
+
+> Move running task(s) to given `cgroups`.
+> More information: <https://manned.org/cgclassify>.
+
+- Move process with pid number 1234 to control group student in cpu hierarchy:
```
Comment (gutjuri) [inline pages/linux/cgclassify.md:6]:
```suggestion
- Move the process with PID 1234 to control group student in the CPU hierarchy:
```

File: pages/linux/cgclassify.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`cgclassify -g {{cpu:student}} {{1234}}`
+
+- Move process with pid number 1234 to control groups based on `/etc/cgrules.conf` configuration file:
```
Comment (gutjuri) [inline pages/linux/cgclassify.md:10]:
```suggestion
- Move the process with PID 1234 to control groups based on the `/etc/cgrules.conf` configuration file:
```

File: pages/linux/cgclassify.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+`cgclassify {{1234}}`
+
+- Move process with pid number 1234 to control group student in cpu hierarchy. The daemon of service `cgred` does not change cgroups of pid 1234 and its children (based on `/etc/cgrules.conf`):
```
Comment (gutjuri) [inline pages/linux/cgclassify.md:14]:
```suggestion
- Move the process with PID 1234 to the control group student in the CPU hierarchy. The daemon of the service `cgred` does not change cgroups of PID 1234 and its children (based on `/etc/cgrules.conf`):
```

**Review by gutjuri — APPROVED** [review]

File: pages/linux/cgclassify.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+- Move the process with PID 1234 to control group student in the CPU hierarchy:
+
+`cgclassify -g {{cpu:student}} {{1234}}`
+
+- Move the process with PID 1234 to control groups based on the `/etc/cgrules.conf` configuration file:
+
+`cgclassify {{1234}}`
+
+- Move the process with PID 1234 to the control group student in the CPU hierarchy. The daemon of the service `cgred` does not change cgroups of PID 1234 and its children (based on `/etc/cgrules.conf`):
```
Comment (kbdharun) [inline pages/linux/cgclassify.md:14]:
```suggestion
- Move the process with a specific PID to the control group student in the CPU hierarchy:

`cgclassify -g {{cpu:student}} {{1234}}`

- Move the process with a specific PID to control groups based on the `/etc/cgrules.conf` configuration file:

`cgclassify {{1234}}`

- Move the process with a specific PID to the control group student in the CPU hierarchy. Note: The daemon of the service `cgred` does not change `cgroups` of the specific PID and its children (based on `/etc/cgrules.conf`):
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by gutjuri — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**